ava-test/cat.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+
+import test from 'ava';
+
+import shell from '..';
+
+test.beforeEach(() => {
+  shell.config.silent = true;
+});
```

Comment:
Is this necessary? In the old code, this is only configured once, not reconfigured before every test. I think this can be simplified to just this:
```
test.before(() => {
    shell.config.silent = true;
});
```
Or even more simply to this:
```
shell.config.silent = true;
```